### PR TITLE
Component Place

### LINKED ISSUE #32: Create a place component.

### PR DESCRIPTION
**Pull request by YufXiao**
Place component include not only component for creating a place but also component for displaying profile of place.

### CODE REVIEW

File: src/styles/_theme.scss [hunk 1]
```diff
@@ -8,8 +8,8 @@
    feel free to play around with these and see what other look you can come up with! :D
 */
 
-$accent: #07b699;
-$accentDark: #068383;
+$accent: #cde9e1e8;
+$accentDark: #bbecdedc;
```
Comment (mystreamer) [inline src/styles/_theme.scss:12]: I like the colour accents 👍

File: src/components/views/PlaceRegister.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            />
+            <FormField
+              label="Nearest To"
+              value={nearestTo}
+              onChange={un => setNearestTo(un)}
+            />
+            <FormField
+              label="Address"
+              value={nearestTo}
+              onChange={un => setNearestTo(un)}
```
Comment (mystreamer) [inline src/components/views/PlaceRegister.js]: This has to of course be its own field!

File: src/components/views/PlaceRegister.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            />
+            <FormField
+              label="Description"
+              value={nearestTo}
+              onChange={un => setNearestTo(un)}
```
Comment (mystreamer) [inline src/components/views/PlaceRegister.js]: This too!

File: src/styles/_theme.scss [hunk 4]
```diff
@@ -8,8 +8,8 @@
...
 $textColor: white;
```
Comment (mystreamer) [inline src/styles/_theme.scss:13]: I would adjust the text color to something darker. It's quite difficult to read...

**Review by mystreamer — COMMENTED** [review]
Looks very nice overall :)

File: src/styles/_theme.scss (hunk 4 above)
Comment (YufXiao) [reply src/styles/_theme.scss:13]: Yes, thanks for reviewing, u can mofidy it as u want.

**Pull request merged**